### PR TITLE
REF: Single source of truth for useIsLargeScreen

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/xiwauguxpw6c22bkdzmfyvjbu4?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1574)

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

File: components/Context/LargeScreenProvider.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
+import React, { createContext, useState, useEffect, useMemo, ReactNode } from 'react';
+import { Dimensions } from 'react-native';
+import { isTablet } from 'react-native-device-info';
+import { isDesktop } from '../../blue_modules/environment';
```
Comment (Overtorment) [inline components/Context/LargeScreenProvider.tsx:4]: feels like we should hide `isTablet` in `/blue_modules/environment` too

Comment (Overtorment) [reply components/Context/LargeScreenProvider.tsx:4]: wdyt?

**Review by Overtorment — CHANGES_REQUESTED** [review]

File: components/Context/LargeScreenProvider.tsx (hunk 1 above)
Comment (marcosrdz) [reply components/Context/LargeScreenProvider.tsx]: addressed

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/i6sjl3ak52hu4bpzfgytv44doe?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1578)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/zdduu4p7msxfsosfzoczfq2qaa?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1579)

**Review by Emmanuelchinyere — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
Wake the fuck up samurai, we have PRs to merge

image

[all PRs for @Overtorment] https://github.com/BlueWallet/BlueWallet/pulls/review-requested/Overtorment

File: blue_modules/environment.ts [hunk 2]
```diff
@@ -1,6 +1,6 @@
 import { isTablet, getDeviceType } from 'react-native-device-info';
```
Comment (Overtorment) [inline blue_modules/environment.ts:1]: isTablet is a function here. Let's export simple Boolean, for consistency?

**Review by Overtorment — CHANGES_REQUESTED** [review]

File: blue_modules/environment.ts (hunk 2 above)
Comment (marcosrdz) [reply blue_modules/environment.ts]: done

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/vjgytk75a3abepldqir34tpozm?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1581)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/rmyha7y2ea77qdlxmkorj3skpu?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1582)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/d7r5t36lpmkrjkydsfqvewp5fq?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1583)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/zcxlwwxjw6ql7zwbqvjbwkq54u?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1584)

**Review by Overtorment — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/gipjznjokvgbdikstkrqyggmrq?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1589)

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/pxl2xl5dtawp5zz75vl6qq765i?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/1597)

**Comment by GladosBlueWallet** [comment]
Unbelievable. You, [subject name here], must be the pride of [subject hometown here]!

**Pull request merged**